<launch>
	<rosparam file="$(find crio_driver)/config/config.yaml" command="load"/>
	
	<arg name="model" default="$(find xacro)/xacro.py '$(find crio_driver)/urdf/robot.urdf'" />
    <param name="robot_description" command="$(arg model)" />

	<node name="crio_driver" 
		pkg="crio_driver"
		type="crio_driver_node"
		output="screen"/>
	
	<node name="controller_spawner" 
		pkg="controller_manager"
		type="spawner"
		args="mobile_base_controller" 
		output="screen"/>
	
</launch>
